plugin/src/cli_utils/clusterinfo/mod.rs

```diff
@@ -0,0 +1,41 @@
+use plugin::ExecuteOperation;
+use crate::cli_utils::CliArgs;
+
+/// CLuster-info operations
+#[derive(clap::Subcommand, Debug)]
+pub(crate) enum Operations {
+    // Gets information of all installed engines.
+    Get(ClusterInfoArg)
+}
+
+#[async_trait::async_trait(?Send)]
+impl ExecuteOperation for Operations {
+    type Args = CliArgs;
+    type Error = Error;
+
+    async fn execute(&self, _cli_args: &CliArgs) -> Result<(), Error> {
+        match self {
+            Operations::Get(_cluster_info_arg) => {
+                todo!("Implementation pending for this command")
+            }
+        }
+    }
+}
+
+/// Arguments required when getting a cluster info.
+#[derive(Debug, Clone, clap::Args)]
+pub struct ClusterInfoArg {
+    /// Helm chart release name.
+    chart: String
+}
```

Comment:
We can remove this for now, as we don't need to use release name and rather this should be done by label selector --> i.e. list by openebs.io/component-name.

We need to add that to mayastor components as well.

Reply:
What would this be used for?

Reply:
There is a command in openebsctl that kind of knows which all components each of the engines have and that command can query through the pod's health to give a unified view of all engine component health.

Now to query the components I am suggesting to not use pod name and figure things out and rather label selectors, using the `openebs.io/component-name` label and list the only selected pods.